gatorgrade.yml

```diff
@@ -0,0 +1,7 @@
+- ? /private/var/folders/y6/yxszy49d45d_vqkhbn13xvfm0000gn/T/pytest-of-anthonygrant-cook/pytest-1/test_generate_config_creates_g0/writing/reflection.md/
```

Comment:
This file should be removed since it doesn’t have any purpose—I think it’s a leftover from a previous test run when the code was not generating the gatorgrade.yml files in the correct location.

Reply:
Thanks for the review. I was able to remove it